src/net/sf/mpxj/primavera/ExternalPredecessorRelation.java

```diff
@@ -0,0 +1,132 @@
+/*
+ * file:       ExternalPredecessorRelation.java
+ * author:     Brandon Herzog
+ * copyright:  (c) Packwood Software 2017
+ * date:       3/20/2017
+ */
+
+/*
+ * This library is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by the
+ * Free Software Foundation; either version 2.1 of the License, or (at your
+ * option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+ * or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public
+ * License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this library; if not, write to the Free Software Foundation, Inc.,
+ * 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+ */
+
+package net.sf.mpxj.primavera;
+
+import net.sf.mpxj.Duration;
+import net.sf.mpxj.RelationType;
+import net.sf.mpxj.Task;
+import net.sf.mpxj.TimeUnit;
+
+/**
+ * This class represents the relationship between two tasks in Primavera when the predecessor
+ * task is not contained in the same project.
+ */
+public final class ExternalPredecessorRelation
```

Comment:
I wonder if we could extend the existing `Relation` class here, and call the new class `ExternalRelation`. We'd set the `sourceTask` to `null`, but provide the `sourceUniqueID` attribute as you have.